src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+    private Set<Object> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("set의 크기를 확인한다.")
+    @Test
+    void set_size() {
+        int expected = 3;
+        assertEquals(expected, numbers.size());
+    }
+
+    @DisplayName("set이 포함하는 값을 확인한다.")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void set_contains(int number) {
+        assertTrue(numbers.contains(number));
```

Comment:
```suggestion
        assertThat(numbers).contains(number);
```

collection의 경우 위와 같이 구현도 가능함